lib/rubocop/cop/lint/void.rb

```diff
@@ -245,15 +245,23 @@ def autocorrect_nonmutating_send(corrector, node, suggestion)
         def entirely_literal?(node)
           case node.type
           when :array
-            node.each_value.all? { |value| entirely_literal?(value) }
+            all_values_entirely_literal?(node)
           when :hash
-            return false unless node.each_key.all? { |key| entirely_literal?(key) }
-
-            node.each_value.all? { |value| entirely_literal?(value) }
+            all_keys_entirely_literal?(node) && all_values_entirely_literal?(node)
+          when :send
```

Comment:
The same applies to the safe navigation operator. Can you add a test case for it as well?

```suggestion
          when :send, :csend
```